hfaerber/Palette_Picker
# Picker component

### What's this PR do? 

- Adds and styles the picker component. 

----------

### How can this be manually tested? 

-`git fetch`
- `git checkout picker-component`
- `npm start`
- Open the component in the browser. Make sure shuffle is working, make sure you are able to post a palette, make sure the error message appears when you try to submit with invalid hex codes or no name. 

---------

### Screenshots: 

<details>

<summary>Picker</summary>

 ![ ](https://user-images.githubusercontent.com/47998896/74195977-95d6d800-4c19-11ea-9500-9df1c27cec80.png)

</details>

----------

### Notes: 

- There is a bug where some of the color panels don't appear after the randomize button is clicked.
